content/docs/5.1/deploy/operate/upgrade.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Upgrade
+description: Learn how to upgrade TiKV using TiUP
+menu:
+    "5.1":
+        parent: Operate TiKV
+        weight: 1
+---
+
+This document is targeted for users who want to upgrade from TiKV 4.0 versions to TiKV 5.0 versions, or from TiKV 5.0 to a later version.
+
+## Preparations
+
+This section introduces the preparation works needed before upgrading your TiKV cluster, including upgrading TiUP and the TiUP Cluster component.
+
+### Step 1: Upgrade TiUP
+
+Before upgrading your TiKV cluster, you first need to upgrade TiUP.
+
+1. Upgrade the TiUP version. It is recommended that the TiUP version is `1.4.0` or later.
+
+    ```bash
+    tiup update --self
+    tiup --version
+    ```
+
+2. Upgrade the TiUP Cluster version. It is recommended that the TiUP Cluster version is `1.4.0` or later.
+
+
+    ```bash
+    tiup update cluster
+    tiup cluster --version
+    ```
+
+### Step 2: Edit TiUP topology configuration file
+
+{{< info >}}
+Skip this step if one of the following situations applies:
+
++ You have not modified the configuration parameters of the original cluster. Or you have modified the configuration parameters using `tiup cluster` but no more modification is needed.
++ After the upgrade, you want to use v5.0's default parameter values for the unmodified configuration items.
+{{< /info >}}
+
+1. Enter the `vi` editing mode to edit the topology file:
+
+    ```bash
+    tiup cluster edit-config <cluster-name>
+    ```
+
+2. Refer to the format of [topology](https://github.com/pingcap/tiup/blob/release-1.4/embed/templates/examples/topology.example.yaml) configuration template and fill the parameters you want to modify in the `server_configs` section of the topology file.
+
+3. After the modification, enter <kbd>:</kbd> + <kbd>w</kbd> + <kbd>q</kbd> to save the change and exit the editing mode. Enter <kbd>Y</kbd> to confirm the change.
```

Comment:
```suggestion
3. Save the changes and exit the editing mode.
```

Reply:
😂 do we really need to tell the audiences how to use vi/vim here?

Reply:
no